src/framapp-polymer-app/login.html

```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../../bower_components/polymerfire/firebase-app">
+<link rel="import" href="../../bower_components/polymerfire/firebase-auth">
+
+<dom-module id="login-component">
+    <template>
+      <style>
+
+      </style>
+      <firebase-app
+        auth-domain="project-5613440220430148247.firebaseapp.com"
+        database-url="https://project-5613440220430148247.firebaseio.com"
+        api-key="AIzaSyBKP4cOP508h0JLKmjFvzJooO0MqV8l4fU">
```

Comment:
please pull master and extend the configuration behavior to contain there settings